caas/kubernetes/provider/modeloperator.go

```diff
@@ -151,13 +157,14 @@ func modelOperatorDeployment(
 	operatorImagePath string,
 	port int32,
 	modelUUID string,
+	serviceName string,
 	volumes []core.Volume,
 	volumeMounts []core.VolumeMount,
 ) (*apps.Deployment, error) {
 
 	moLabels := modelOperatorLabels(operatorName)
 
-	jujudCmd := fmt.Sprintf("$JUJU_TOOLS_DIR/jujud model --model-uuid=%s", modelUUID)
+	jujudCmd := fmt.Sprintf("$JUJU_TOOLS_DIR/jujud model --model-uuid=%s --logging-config=\"<root>=DEBUG\"", modelUUID)
```

Comment:
logging config should come from the model config setting, not hard coded
There's a logger worker which handles this. If not already part of the model operator manifold, it needs to be added

Reply:
Done & Done